db/db_bloom_filter_test.cc

```diff
@@ -146,6 +148,172 @@ TEST_P(DBBloomFilterTestDefFormatVersion, KeyMayExist) {
       ChangeOptions(kSkipPlainTable | kSkipHashIndex | kSkipFIFOCompaction));
 }
 
+TEST_F(DBBloomFilterTest, PartitionedMultiGet) {
+  for (bool use_prefix : {false, true}) {
```

Comment:
Suggest parameterizing the test with ```use_prefix```. It would make it a little easier to debug a test failure.